drivers/alsamidi/midi_driver_alsamidi.cpp

```diff
@@ -84,8 +102,23 @@ void MIDIDriverALSAMidi::thread_func(void *p_udata) {
 						}
 						expected_size = get_message_size(byte);
 					}
-
-					if (bytes < 256) {
+					
```

Comment:
It doesn't seem visible on the GitHub diff preview, but there seems to be trailing whitespace on this line.

Reply:
I've cleaned the whitespace. Also, I've switched to another way of dealing with SysExes, and left a comment to explain its reasoning.